openedx/features/course_experience/templates/course_experience/course-sock-fragment.html

```diff
@@ -16,7 +16,7 @@
         %endif
     >
         <button type="button" class="btn btn-brand focusable action-toggle-verification-sock">
-            Learn About Verified Certificate
+            Learn About Verified Certificates
```

Comment:
The AC read "Learn about verified certificates".  Just double-checking that:
1. We want the old capitalization and
2. That we don't want to end with a period.
FYI: @marcotuts @catong

Reply:
Keeping initial caps, and no period looks good to me. 👍